_episodes/02-numpy.md

```diff
@@ -665,38 +665,38 @@ which is the average inflammation per patient across all days.
 > with NumPy.
 >
 > The `numpy.diff()` function takes an array and returns the differences
-> between two successive values. First we consider a one-dimensional
-> array of length 5. This could be part of some row `i` of our inflammation data,
-> i.e. `row_start = data[i,:5]`.
->
+> between two successive values. Let's use it to examine the changes
+> each day across the first week of patient 3 from our inflammation dataset.
+> 
 > ~~~
-> row_start = numpy.array([ 0,  2,  5,  9, 14])
+> patient3_week1 = data[3, :7] 
 > ~~~
 > {: .language-python}
 >
-> Calling `numpy.diff(row_start)` would do the following calculations
+> Calling `numpy.diff(patient3_week1)` would do the following calculations
 >
 > ~~~
-> [ 2 - 0, 5 - 2, 9 - 5, 14 - 9 ]
+> [ 0 - 0, 2 - 0, 0 - 2, 4 - 0, 2 - 4, 2 - 2 ]
 > ~~~
 > {: .language-python}
 >
-> and return the 4 difference values in a new array.
+> and return the 6 difference values in a new array.
 >
 > ~~~
-> numpy.diff(row_start)
+> numpy.diff(patient3_week1)
 > ~~~
 > {: .language-python}
 >
 > ~~~
-> array([2, 3, 4, 5])
+> array([ 0,  2, -2,  4, -2,  0])
```

Comment:
When I run the code, I get the output with dots like:

`array([ 0.,  2., -2.,  4., -2.,  0.])`

so perhaps we should include the dots to match what people will actually see. (I know I am running an older version of Python 3 and probably `numpy` though, so if the output you already have here is what you get in actuality, please disregard this comment).